template/6-non_functional_requirements.md

```diff
@@ -2,22 +2,46 @@
+
+### Location Policy
+
+The primary privacy consideration for Proxima concerns the management of user location data. Recognizing the sensitivity of this information, we have established a strict policy regarding its usage.
+
+Proxima will not store any user location data on the backend. The only location data stored in the backend are the locations of posts. While these locations are initially linked to a user at the time of posting, they quickly become non-relevant as users move to different places. This approach ensures that users' location privacy is maintained while still enabling the core functionalities of the application.
+
+### Moderation
+
+Moderation is a crucial aspect of any public social media platform, and our application is no exception. Our policy regarding moderation will be based on a reporting system complemented by AI analysis and human judgment.
+
+If content is reported by multiple users, it will first be evaluated by an AI system to determine its appropriateness. If the AI model cannot fully decide on the nature of the content, human moderators will be engaged to make the final decision on whether to remove the content or not. This combined approach ensures that moderation is efficient, accurate, and fair, maintaining a safe and respectful environment for all users.
 
-- Privacy
-- Moderation
-- Authentication
 
 ## Adoptions, Scalability and Availability
 
-*What kind of traffic patterns do you expect to see?*
+### Adoption
+
+1. **Quick Engagement**: Upon the initial launch of the app, users should be able to view and post content with minimal steps to reduce conversion costs and facilitate quick engagement.
+2. **Challenge Tour**: When accessing the challenges page for the first time, users should receive a brief tour explaining how the challenges work and the logic behind Centauri points.
+3. **Location Awareness**: Users should be able to view nearby content on a map immediately, highlighting the location-based features of the application.
+
+### Scalability
+
+1. **High User Capacity**: The application must scale well to accommodate a high number of users and a large volume of content, ensuring seamless performance as the user base grows.
+2. **Content Density Performance**: It must also maintain good performance and responsiveness in high content density areas, such as big towns, monuments, and concerts, to provide a smooth user experience in these environments.
+3. **Feature Flexibility**: The backend should be designed to easily support the addition of new features over time, ensuring the application can evolve and adapt to meet changing user needs and technological advancements.
```

Comment:
I think you should also mention the fact that we want the backend to be designed in such a way that is backward compatibility (never requiring a database wipe). This might be a bit repetitiive with `Update Pipeline` right after however.

Moreover, this is quite vague. I think the PRD is as vague as that, but I just want to be sure this is what is expected. What do you think?

Reply:
That's a good idea indeed !
I add backward compatibility part in 57704e8f773346cc4f886536115f0dc9a3b25f2c.
Thanks for the suggestion :+1: 

Yes, this is pretty vague on purpose. The objective is only to give rough guidelines without specifying the exact implementation. This is also to keep the PRD synthetic.